### ISSUE TITLE
About the idol.sankaku new API

### ISSUE THREAD

**Issue by 95e2** [Bionus/imgbrd-grabber#1294]
I found the new API for sankaku from Android App

API BaseURL: iapi.sankakucomplex.com
User-Agent: 'SCChannelApp/3.0 (Android; idol)'

**Comment by MrAndre96** [comment]
You are able to change the user-agent yourself inside the settings. After that this source will work fine

**Comment by Bionus** [comment]
Yes, it's also the same API used for https://beta.sankakucomplex.com/ (**NSFW**), excepts for `chan` it uses `capi{-beta}` and for `idol` it uses `iapi{-beta}` .
